plugins/chat/assets/javascripts/discourse/components/chat-thread.js

```diff
@@ -253,11 +253,11 @@ export default class ChatThreadPanel extends Component {
     return this.chatApi
       .sendMessage(this.channel.id, {
         message: message.message,
-        in_reply_to_id: message.inReplyTo?.id,
+        in_reply_to_id: message.thread.originalMessage.id,
```

Comment:
I guess technically this can just always be `null` here, since you are never replying to a message in a thread (at this point in time) and we don't have plans to in the short term. Passing the `thread_id` is enough